src/components/settingsdialog/AppearanceSettings.tsx

```diff
@@ -21,6 +21,7 @@ const AppearanceSettings = (): JSX.Element => {
 		hideSelfView,
 		controlButtonsBar
 	} = useAppSelector((state) => state.settings);
+	const browser = useAppSelector((state) => state.me.browser);
```

Comment:
Make the same variable here, `isMobile`, and check that down below instead.

Reply:
Done